Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -3202,17 +3202,17 @@ bool CStaticFunctionDefinitions::SetVehicleLightState(CClientEntity& Entity, uns
     return false;
 }
 
-bool CStaticFunctionDefinitions::SetVehiclePanelState(CClientEntity& Entity, unsigned char ucPanel, unsigned char ucState)
+bool CStaticFunctionDefinitions::SetVehiclePanelState(CClientEntity& Entity, unsigned char ucPanel, unsigned char ucState, bool spawnFlyingComponent, bool breakGlass)
 {
-    RUN_CHILDREN(SetVehiclePanelState(**iter, ucPanel, ucState))
+    RUN_CHILDREN(SetVehiclePanelState(**iter, ucPanel, ucState, spawnFlyingComponent, breakGlass))
 
     if (IS_VEHICLE(&Entity))
     {
         CClientVehicle& Vehicle = static_cast<CClientVehicle&>(Entity);
 
         if (ucPanel < 7)
         {
-            Vehicle.SetPanelStatus(ucPanel, ucState);
+            Vehicle.SetPanelStatus(ucPanel, ucState, spawnFlyingComponent, breakGlass);
```

Comment:
This PR is not aimed at refactoring, so in this case early-return is already beyond the scope of this PR

Reply:
I think u can add the validation to fix that issue too I do think anyone will fix this later on

Reply:
it does work so the issue is just return true rather than false in that code

```c++
bool CStaticFunctionDefinitions::SetVehiclePanelState(CClientEntity& Entity, unsigned char ucPanel, unsigned char ucState, bool spawnFlyingComponent, bool breakGlass)
{
    RUN_CHILDREN(SetVehiclePanelState(**iter, ucPanel, ucState, spawnFlyingComponent, breakGlass))

    if (IS_VEHICLE(&Entity))
    {
        CClientVehicle& Vehicle = static_cast<CClientVehicle&>(Entity);

        if (ucPanel < 7)
        {
            Vehicle.SetPanelStatus(ucPanel, ucState, spawnFlyingComponent, breakGlass);
            return true;
        }
    }

    return false;
}
```